src/main/java/javaslang/collection/Seq.java

```diff
@@ -97,6 +97,14 @@
  */
 public interface Seq<T> extends Traversable<T>, IntFunction<T> {
 
+    static <T> Seq<T> ofAll(Traversable<T> traversable) {
```

Comment:
Awesome! Overall it now looks really great.

One more thing: let's relax `ofAll(Traversable)` and do the following:

```java
static <T> Seq<T> ofAll(java.lang.Iterable<? extends T> elements) { ... }
```

Thx!